AllInOne.cs

```diff
@@ -980,24 +1008,23 @@ public void ConditionalRef()
 
     public void LeadingSeparator()
     {
-        var res = 0
-        + 123      // permitted in C# 1.0 and later
-        + 1_2_3    // permitted in C# 7.0 and later
-        + 0x1_2_3  // permitted in C# 7.0 and later
-        + 0b101    // binary literals added in C# 7.0
-        + 0b1_0_1  // permitted in C# 7.0 and later
-
-        // in C# 7.2, _ is permitted after the `0x` or `0b`
-        + 0x_1_2   // permitted in C# 7.2 and later
-        + 0b_1_0_1 // permitted in C# 7.2 and later
```

Comment:
If we realign the comments and then re-run, would we lose the alignment again? Or is this only sensitive to the improper leading whitespace